db/table_properties_collector.h

```diff
@@ -108,4 +110,65 @@ class UserKeyTablePropertiesCollectorFactory
   std::shared_ptr<TablePropertiesCollectorFactory> user_collector_factory_;
 };
 
+// When rocksdb creates a newtable, it will encode all "user keys" into
+// "internal keys". This class collects min/max timestamp from the encoded
+// internal key when Add() is invoked.
+//
+// @param cmp the Comparator to compare timestamp of key by the
+//     Comparator::CompareTimestamp function.
```

Comment:
Not sure, but the indentation here seems weird.

Reply:
fixed